cmd/admin-revoker/main.go

```diff
@@ -221,6 +372,108 @@ func (rc revocationCodes) Len() int           { return len(rc) }
 func (rc revocationCodes) Less(i, j int) bool { return rc[i] < rc[j] }
 func (rc revocationCodes) Swap(i, j int)      { rc[i], rc[j] = rc[j], rc[i] }
 
+// loadPrivateKey decodes and parses a private key from the provided
+// `keyContents` and returns the private key as `crypto.Signer`. `keyContents`
+// is expected to be a PEM formatted RSA or ECDSA private key in a PKCS #1,
+// PKCS# 8, or SEC 1 container.
+func loadPrivateKey(keyContents []byte) (crypto.Signer, error) {
```

Comment:
This function largely duplicates `loadSigner` in issuance/issuance.go. Maybe factor both out into a `privatekey` package with a single method `Load()`?

Reply:
This is a great idea. I went ahead and consolidated the logic into a new privatekey package. I was also able to move the embedded PublicKey verification methods into this package as well. As a bonus I threw in some coverage for the loading logic that wasn't there before.